Doc/howto/clinic.rst

```diff
@@ -1891,3 +1891,82 @@ blocks embedded in Python files look slightly different.  They look like this:
     #[python start generated code]*/
     def foo(): pass
     #/*[python checksum:...]*/
+
+
+How to deprecate positional use of optional parameters
+------------------------------------------------------
+
+Argument Clinic provides syntax that makes it possible to generate code that
+deprecates positional use of optional parameters.
+For example, say we've got a module level function :py:func:`!foo.myfunc` that
+takes three :class:`int` parameters:
```

Comment:
```suggestion
has three :class:`int` :term:`parameters <parameter>`:
```

Reply:
Hm, I prefer "takes" here. (But I agree with linking to the glossary definition of "parameter".)

Reply:
Function *has* parameters (specified in function definition). It *takes* arguments (when called). If I understand this correctly.